pbft/database_pbft_service.cpp

```diff
@@ -76,7 +76,7 @@ database_pbft_service::apply_operation(const std::shared_ptr<bzn::pbft_operation
 
 
 bool
-database_pbft_service::apply_operation_now(const bzn_envelope& msg, std::shared_ptr<bzn::session_base> session)
+database_pbft_service::apply_operation_now(const bzn_envelope& msg, std::shared_ptr<bzn::session_base> /*session*/)
```

Comment:
You will break quick read here by not sending in session.

Reply:
OK, putting session back in this case. Could this have been caught with a unit test?

Reply:
There's already a test for handle_request. 
The tests for this are ensuring that it's called from the db service. It just passes the pointer along WHICH will never be null.